airflow/migrations/versions/e38be357a868_update_schema_for_smart_sensor.py

```diff
@@ -50,6 +51,11 @@ def sa_timestamp():  # noqa: D103
 def upgrade():  # noqa: D103
 
     conn = op.get_bind()
+    inspector = Inspector.from_engine(conn)
+    tables = inspector.get_table_names()
+    if "sensor_instance" in tables:
```

Comment:
nit: good to use single quotes to keep consistent with the whole script? The same for the change in `downgrade()`

Reply:
Done

Reply:
👍